### PR TITLE
remove old (unused) usages of heartbeat monitor

### PR DESCRIPTION
**Pull request by cgardens**
## What
* At some point we had a working heartbeat monitor that tracked that the source was regularly sending records to the platform. 
* At some point, it got partially disconnected. In other words, it doesn't work at all, but it still exists in the code.
* This PR removes the old, unused usages.
* Next PR will plug in a new heartbeat monitor

@davinchia a sanity check here would be appreciated.

### CODE REVIEW

File: airbyte-commons-worker/src/main/java/io/airbyte/workers/internal/DefaultAirbyteSource.java [hunk 1]
```diff
@@ -87,7 +72,6 @@ public void start(final WorkerSourceConfig sourceConfig, final Path jobRoot) thr
     logInitialStateAsJSON(sourceConfig);
 
     messageIterator = streamFactory.create(IOs.newBufferedReader(sourceProcess.getInputStream()))
-        .peek(message -> heartbeatMonitor.beat())
```
Comment (cgardens) [inline airbyte-commons-worker/src/main/java/io/airbyte/workers/internal/DefaultAirbyteSource.java:90]: we are triggering the heartbeat here, but never checking it anywhere else.

File: airbyte-commons-worker/src/test/java/io/airbyte/workers/WorkerUtilsTest.java [hunk 2]
```diff
@@ -5,125 +5,22 @@
...
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
 class WorkerUtilsTest {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(GentleCloseWithHeartbeat.class);
-
-  @Nested
-  class GentleCloseWithHeartbeat {
```
Comment (cgardens) [inline airbyte-commons-worker/src/test/java/io/airbyte/workers/WorkerUtilsTest.java:42]: i don't think this is testing anything that is still used. but @davinchia  this is where i feel least sure. it looks like in the original orchestrator implementation some of this code got copied over. but i think it's not used.

File: airbyte-commons-worker/src/test/java/io/airbyte/workers/internal/DefaultAirbyteSourceTest.java [hunk 3]
```diff
@@ -165,21 +160,16 @@ void testSuccessfulLifecycle() throws Exception {
...
     when(process.getErrorStream()).thenReturn(new ByteArrayInputStream(("rewq").getBytes(StandardCharsets.UTF_8)));
 
-    when(heartbeatMonitor.isBeating()).thenReturn(true).thenReturn(false);
-
-    final AirbyteSource source = new DefaultAirbyteSource(integrationLauncher, streamFactory,
-        heartbeatMonitor);
+    final AirbyteSource source = new DefaultAirbyteSource(integrationLauncher, streamFactory);
     source.start(SOURCE_CONFIG, jobRoot);
 
-    final List<AirbyteMessage> messages = Lists.newArrayList();
```
Comment (cgardens) [inline airbyte-commons-worker/src/test/java/io/airbyte/workers/internal/DefaultAirbyteSourceTest.java:179]: this array isn't being used.

**Comment by davinchia** [comment]
looking

File: airbyte-commons-worker/src/test/java/io/airbyte/workers/WorkerUtilsTest.java (hunk 2 above)
Comment (davinchia) [reply airbyte-commons-worker/src/test/java/io/airbyte/workers/WorkerUtilsTest.java:42]: your comment looks right to me. 

best way to confirm this is to make sure the acceptance tests pass. I've fixed the PMD error so we can run the full build.

File: airbyte-commons-worker/src/main/java/io/airbyte/workers/internal/DefaultAirbyteSource.java [hunk 4]
```diff
@@ -57,16 +54,8 @@ public DefaultAirbyteSource(final IntegrationLauncher integrationLauncher) {
   }
```
Comment (davinchia) [inline airbyte-commons-worker/src/main/java/io/airbyte/workers/internal/DefaultAirbyteSource.java:54]: @cgardens this is needed for the abstract source tests - did not look closely why. mainly wanted to get the build working.

**Comment by davinchia** [comment]
Build is working and I'm confident this change is good.

**Comment by cgardens** [comment]
Thanks, Davin! Can you approve please then?

**Review by davinchia — APPROVED** [review]

**Pull request merged**